schemas/2015-06-01/Microsoft.KeyVault.json

```diff
@@ -0,0 +1,209 @@
+                    "description": "Microsoft.KeyVault/vaults: The tenant ID of the Azure Active Directory Tenant containing the objectId in this access policy."
+                },
+                "objectId": {
+                    "type": "string",
+                    "pattern": "^[0-9a-fA-F]{8}(-[0-9a-fA-F]{4}){3}-[0-9a-fA-F]{12}$",
+                    "description": "Microsoft.KeyVault/vaults: The object ID of the Azure Active Directory object to apply this access policy to.  For example, the object ID of a Service Principal, or of a User Principal."
+                },
+                "permissions": {
+                    "$ref": "#/definitions/permissions"
+                }
+            },
+            "required": [
+                "tenantId",
+                "objectId",
+                "permissions"
+            ]
+        },
+        "sku": {
+            "type": "object",
+            "properties": {
+                "name": {
+                    "enum": [
+                        "standard",
+                        "premium"
+                    ],
+                    "description": "Microsoft.KeyVault/vaults: The service tier of KeyVault to use.  Standard supports secrets and software-protected keys.  Premium adds support for HSM-protected keys."
+                },
+                "family": {
+                    "enum": [
+                        "A"
+                    ],
+                    "description": "Microsoft.KeyVault/vaults: The SKU family to use."
+                }
+            },
+            "required": [
+                "family",
+                "name"
+            ]
+        },
+        "vaults": {
+            "type": "object",
+            "properties": {
+                "type": {
+                    "enum": [
+                        "Microsoft.KeyVault/vaults"
+                    ]
+                },
+                "apiVersion": {
+                    "enum": [
+                        "2014-12-19-preview",
+                        "2015-06-01"
+                    ]
+                },
+                "properties": {
+                    "type": "object",
+                    "properties": {
+                        "sku": {
+                            "$ref": "#/definitions/sku"
+                        },
+                        "tenantId": {
+                            "type": "string",
+                            "pattern": "^[0-9a-fA-F]{8}(-[0-9a-fA-F]{4}){3}-[0-9a-fA-F]{12}$",
+                            "description": "Microsoft.KeyVault/vaults: The tenant ID of the Azure Active Directory Tenant to use for authorization."
+                        },
+                        "accessPolicies": {
+                            "type": "array",
+                            "items": {
+                                "$ref": "#/definitions/accessPolicy"
+                            },
+                            "enabledForDeployment": {
```

Comment:
D'oh.  Yes.  I messed this up when I was rearranging code.  Good catch - thank you.